changelogs/unreleased/gh-7747-seqscan.md

```diff
@@ -3,3 +3,4 @@
 * Introduced a new `SEQSCAN` keyword for usage in scanning `SELECT` queries.
   A new session setting `sql_seq_scan` can be used to allow or restrict scanning
   queries without `SEQSCAN` (gh-7747).
+* Compat option sql_seq_scan_default was introduced (gh-8096).
```

Comment:
There's no need to add a new entry for this change or mention the ticket. Let's change to this:

```
* Introduced the new keyword `SEQSCAN` for SQL `SELECT` queries. You may now
  use a scanning SQL `SELECT` query without the `SEQSCAN` keyword only if the
  `sql_seq_scan` session setting is set to true. The default session setting
  value is is controlled by the new `compat` option `sql_seq_scan_default`
  (gh-7747).
```